### DISCUSSION TITLE
Difference between `delete_one` and `delete` on embeds_many association?

### DISCUSSION THREAD

**Discussion by imechemi** [mongodb/mongoid#5538] [Q&A]
I noticed that `delete_one`  embeds_many association does not persist the changes automatically whereas the `delete` automatically persist. So I have two questions

1. Is my observation correct? 
2. Are there any other difference that I should be cautious of? 

It would be great to get clarification on it. Thank you 😇

**Comment by alexbevi** [comment]
Hi @imechemi, thanks for the question! Can you clarify which version of Mongoid you're using. I'm assuming this behavior has not changed but we want to ensure we're answering the right question for the version being used.

**Reply by comandeo** [reply]
Hi @imechemi, thank you for reporting this. This is indeed an unexpected behaviour. We've opened a ticket to investigate and fix the problem - https://jira.mongodb.org/browse/MONGOID-5560

**Reply by imechemi** [reply]
@comandeo  I actually needed an API to update embedded association without persisting it automatically. If this API would be updated, what's the suggested way to remove or replace embedded association without persisting.

**Reply by imechemi** [reply]
@alexbevi I am using `7.4.x` currently.
